torch/csrc/jit/codegen/cuda/ir_iostream.cpp

```diff
@@ -44,13 +43,16 @@ void IRPrinter::handle(const Expr* e) {
 void IRPrinter::printHeader(Fusion* fusion, const std::string& kernel_name_) {
   os << "__global__ void " << kernel_name_ << "(";
 
-  std::deque<Val*> vals;
-  for (decltype(fusion->nInputs()) i{0}; i < fusion->nInputs(); i++)
-    vals.push_back(fusion->input(i));
-  for (decltype(fusion->nOutputs()) i{0}; i < fusion->nOutputs(); i++)
-    vals.push_back(fusion->output(i));
+  std::vector<Val*> values;
```

Comment:
Nit: I typically use vals as it's literally multiple `Val` nodes

Reply:
Having two local variables named `val` and `vals` seems cruel to human readers.

Reply:
🤷‍♂️ maybe I'm just cruel then, but when having multiples of a thing I like to just append the s to the thing

Reply:
Ok, renamed back to `vals`.